rasa/telemetry.py

```diff
@@ -573,6 +574,11 @@ def strip_sensitive_data_from_sentry_event(
     for value in event.get("exception", {}).get("values", []):
         for frame in value.get("stacktrace", {}).get("frames", []):
             frame["abs_path"] = ""
+
+            if "rasa_sdk/executor.py" in frame["filename"]:
+                # this looks a lot like an exception in the SDK and hence custom code
+                # no need for us to deal with that
+                return None
```

Comment:
looks more like a filter than stripping sensitive info, but arguable 👍

Reply:
well, but the intent is to remove sensitive info 😉